src/network/package.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "../util/util.h"
+#include "../math/types.h"
+#include "../event.h"
+
+enum class PackageType {
+    A,
+    B,
+    EVENT,
+
+    // Types below this line should not be creatable and thus not shown in the package type list.
+
```

Comment:
Introduce an enum that says how many names should exist, so we can enforce it when we compile the program?